pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -28,8 +30,24 @@
 }:
 
 let
-  version = "5.9.6.2225";
+  inherit (stdenv.hostPlatform) system;
+  throwSystem = throw "Unsupported system: ${system}";
+
+  version = {
+    aarch64-darwin = "5.10.1.5839";
+    x86_64-darwin = "5.10.1.5839";
+    x86_64-linux = "5.9.6.2225";
```

Comment:
Let's add a comment here explaining why do we use different versions for different systems, with a link to https://github.com/NixOS/nixpkgs/pull/166085 .

Also, according to [this comment](https://github.com/NixOS/nixpkgs/pull/166085#issuecomment-1106492619) there's a 5.10.3 now.